M2/Macaulay2/packages/Macaulay2Doc/doc12.m2

```diff
@@ -629,37 +629,34 @@ document {
      Key => {ScriptedFunctor,
 	  (symbol^, ScriptedFunctor, Thing),
 	  (symbol_, ScriptedFunctor, Thing),
-	  (symbol SPACE, ScriptedFunctor, Thing)},
+	  Functor, (symbol SPACE, Functor, Thing)},
```

Comment:
You've documented "Functor" by simply pointing to the documentation for "ScriptedFunctor", but that documentation node doesn't mention "Functor", nor does it explain the difference between the two.  Why not make a new node for "Functor" and explain it, really?  Include code showing an example of a functor that's not a scripted functor.

Reply:
Is there really a point for separating Functor and ScriptedFunctor? Given that not all functors are scripted, I can just rename ScriptedFunctor to Functor if you prefer.

Reply:
I documented Functor for now, but long term, it might be simpler to eliminate ScriptedFunctor.

Reply:
> Is there really a point for separating Functor and ScriptedFunctor? 

It's your pull request that introduces Functor, so you must have thought there was a point -- probably because `f_*` has neither a subscript nor a superscript.

Reply:
Yes, I introduced `Functor` for mathematical reasons rather than syntactic reasons, and made `ScriptedFunctor` inherit from it for semantic reasons. But I do not care for the term `ScriptedFunctor` and would rather they be removed in a future PR.